libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/MapboxOnboardRouter.kt

```diff
@@ -58,26 +58,32 @@ class MapboxOnboardRouter : Router {
     }
 
     override fun getRoute(
-        routeOptions: RouteOptionsNavigation,
+        routeOptions: RouteOptions,
         callback: Router.Callback
     ) {
+
+        val origin = routeOptions.coordinates().first()
```

Comment:
As above, we should assert that there are at least 2 points and throw a meaningful error if needed.

Reply:
done